formatters/junitFormatter.js

```diff
@@ -0,0 +1,70 @@
+"use strict";
```

Comment:
So tslint needs cjs file as input. When I tried to pass it ts it balked at me.

I think the magic I'm missing is a prepublish task in the package.json https://docs.npmjs.com/misc/scripts#common-uses

So the cjs wouldn't be in source control but would be in npm.